pymedphys/tests/delivery/test_exported_object.py

```diff
@@ -1,3 +1,18 @@
+# Copyright (C) 2019 Simon Biggs
```

Comment:
If you added the copyright in 2020 but the body of work was done in 2019, maybe it should indicate either 2020, or 2019 and 2020?

Reply:
Yeah, I don't think it'd hurt here to have 2019-2020.